_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 'Bitcoin Optech Newsletter #103'
+permalink: /en/newsletters/2020/06/24/
+name: 2020-06-24-newsletter
+slug: 2020-06-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a newly published fee ransom attack against
+LN users, links to continued discussion about the attack against LN
+atomicity, and shares a reminder about collision attacks on
+RIPEMD160-based addresses in multiparty protocols.  Also included are
+our regular sections with popular questions and answers from the Bitcoin
+StackExchange, a list of releases and release candidates published this
+week, and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **LN fee ransom attack:** René Pickhardt [publicly
+  disclosed][pickhardt post] a vulnerability to the Lightning-Dev
+  mailing list that he had previously privately disclosed to LN
+  implementation maintainers almost a year ago.  In the current LN
+  protocol, the party who initiates opening a channel must commit to
+  paying any onchain transaction fees that will be required to later
+  close that channel.  The party paying the fees also gets to choose the
+  feerate used---but the security of both parties depends on choosing an
+  appropriate feerate, so the counterparty may close the channel if they
```

Comment:
I think this should be 'refuse to open the channel' rather than 'close the channel'. BOLT 2 says "The receiving node MUST fail the channel if [...] it considers feerate_per_kw too small for timely processing or unreasonably large."

Reply:
That's a good point and I'll try to rephrase to be more clear, but I think the issue is more with variable feerates.  If Alice opens a channel to Bob when 10 nBTC/vbyte is sufficient to get a commitment transaction confirmed quickly and Alice agrees to pay 50 nBTC/vbyte (the 5x example), the channel will be opened and Alice can start making payments.  But if feerates then rise to 30 nBTC/vbyte being necessary, Bob's node has to consider publishing the commitement before the the safety margin of (50 - 30 = 20 nBTC/vbyte) disappears.  See [this section of BOLT2](https://github.com/lightningnetwork/lightning-rfc/blob/master/02-peer-protocol.md#updating-fees-update_fee) for details (which is the section where the 5x example comes from).